documentation/latest/dev/guide.md

```diff
@@ -4,171 +4,38 @@ title: Developer guide
-### Troubleshooting
-
-**kubectl wait timeouts:**
-
-If timeout, debug the deployment logs by running:
-
-```shell 
-kubectl logs deploy/flotta-controller-manager -n flotta
-```
-
-**Waiting for edge device timeouts in integration tests**:
-
-Debug the edge device container by executing shell inside the container:
-
-```shell
-docker exec -it edgedevice1 journalctl -u yggdrasild.service
-```
-
-**Error: "No such image: quay.io/project-flotta/edgedevice:latest"**
-
-This error happens when KinD does not have the image stored in the control plane. To solve it, use
-
-```shell
-docker pull quay.io/project-flotta/edgedevice:latest
-kind load docker-image quay.io/project-flotta/edgedevice:latest
-```
-
-command to upload the missing image.
-
-**How can I see the running containers in the device?**
-
-To list the podman managed pods running in a device, run this command in the target device:
-
-```shell
-docker exec -it edgedevice1 machinectl shell -q flotta@.host /usr/bin/podman pod ps
+mvn integration-test -P integrationtest
 ```
 
-or
+## Details
+_Parodos_ treats integration tests as unit tests and executes them independently using the Maven _Failsafe_ plugin instead of the classic _Surefire_ plugin.
 
-```shell
-machinectl shell -q flotta@.host /usr/bin/podman pod ps
-```
+The Maven _Surefire_ and _Failsafe_ plugins are two widely used plugins for executing tests during a Maven build. These two plugins differ in the build lifecycle phase in which they operate. While the _Surefire_ plugin runs tests during the test phase, the _Failsafe_ plugin executes tests during the integration-test phase.
 
-if you're running it inside the container.
+The _Failsafe_ plugin is designed to execute integration tests that require more elaborate setup, such as tests that rely on a running application server or database connection. Integration tests run in the same JVM instance as the application, and the plugin includes a default naming convention for identifying integration test classes. _Failsafe_ generates reports that provide insight into the overall health of the application, as well as any integration issues that may exist.
 
-**How can I see the logs produced by the flotta user in the device?**
+To facilitate running integration tests during the Maven build process, _Parodos_ defines a Maven profile named `integrationtest`. The command to execute the integration tests is:
 
-When running podman in rootless mode, the logs from the agent and podman are captured for the flotta user running in the device. Run the following command to see the journalctl logs:
-
-```shell
-machinectl shell -q flotta@.host /usr/bin/journalctl --user
+```bash
+mvn integration-test -P integrationtest
 ```
+You can also specify the server IP and server port by setting the environment variables `SERVERIP` and `SERVERPORT`. By default, the values are `localhost` and `8080`, respectively. For instance, you can run the following command to specify custom values for the server IP and port:
```

Comment:
I'd suggest using snake-case for variable names for readability: SERVER_PORT and SERVER_IP